protocol/x/prices/types/message_create_oracle_market_test.go

```diff
@@ -0,0 +1,108 @@
+package types_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/dydxprotocol/v4-chain/protocol/testutil/constants"
+	types "github.com/dydxprotocol/v4-chain/protocol/x/prices/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMsgCreateOracleMarket_GetSigners(t *testing.T) {
+	msg := types.MsgCreateOracleMarket{
+		Authority: constants.AliceAccAddress.String(),
+	}
+	require.Equal(t, []sdk.AccAddress{constants.AliceAccAddress}, msg.GetSigners())
+}
+
+func TestMsgCreateOracleMarket_ValidateBasic(t *testing.T) {
+	validExchangeConfigJson := "{\"exchanges\":[{\"exchangeName\":\"Binance\",\"ticker\":\"\\\"BTCUSDT\\\"\"}]}"
+	tests := []struct {
+		desc        string
+		msg         types.MsgCreateOracleMarket
+		expectedErr string
+	}{
+		{
+			desc:        "Empty authority",
+			msg:         types.MsgCreateOracleMarket{},
+			expectedErr: "authority cannot be empty",
+		},
+		{
+			desc: "Valid MsgCreateOracleMarket",
+			msg: types.MsgCreateOracleMarket{
+				Authority: "test",
+				Params: types.MarketParam{
+					Pair:               "BTC-USD",
+					MinExchanges:       1,
+					MinPriceChangePpm:  1_000,
+					ExchangeConfigJson: validExchangeConfigJson,
+				},
+			},
+			expectedErr: "",
```

Comment:
In this case, you shouldn't need to explicitly populate `expectedErr`, because `""` is the zero value.

Reply:
My personal preference in these type of tests are to explicitly put literal values that are default values, e.g. `expectedErr == ""`, `expectedBalance == 0` for readability. Let me know if you have strong preference otherwise

Reply:
I think that seems fine. My only remaining potential concern is inconsistency in style across the codebase, but it sounds like we may already have that, so it wouldn't make sense to address that here.